Error in Getting Started documentation

**Describe the bug**
When following the instructions in https://github.com/ggez/ggez/blob/master/docs/guides/HelloGgez.md I encountered the following error during the `cargo run` for the final step
```
error[E0609]: no field `time` on type `&mut ggez::Context`
  --> src/main.rs:10:23
   |
10 |         self.dt = ctx.time.delta();
   |                       ^^^^ unknown field
   |
   = note: available fields are: `filesystem`, `timer_context`, `audio_context`, `keyboard_context`, `mouse_context` ... and 2 others
```

**To Reproduce**
Follow the instructions found in https://github.com/ggez/ggez/blob/master/docs/guides/HelloGgez.md

**Expected behavior**
The `time` field should be available on `ctx` or else the documentation should be updated (it specifies version `0.7` but the code seems to be using version `0.8`).

**Hardware and Software:**
 - ggez version: 0.7.0
 - OS: MacOS 12.4
 - Graphics card: N/A
 - Graphics card drivers: N/A

--- a/RELEASE_PROCEDURES.md
+++ b/RELEASE_PROCEDURES.md
@@ -8,6 +8,7 @@ API.  (Doing this for packages that aren't publically exposed is PROBABLY okay..
  * Make sure all unit tests pass
  * Test all examples
  * Read all docs (ideally in rendered form)
+ * Also read (and ideally test) all docs and guides inside the `docs` folder! (check that links are up to date, etc.)
  * Make sure website is updated and in-sync
  * rustfmt
  * clippy

--- a/docs/BuildingForEveryPlatform.md
+++ b/docs/BuildingForEveryPlatform.md
@@ -37,7 +37,7 @@ wget https://raw.githubusercontent.com/ggez/ggez/master/resources/LiberationMono
 Then edit your `Cargo.toml` with your favorite super duper editor and under `[dependencies]` add:
 
 ```
-ggez = "0.7"
+ggez = "0.8.0-rc0"
 glam = { version = "0.20", features = ["mint"] }
 ```
 

--- a/docs/guides/GenerativeArt.md
+++ b/docs/guides/GenerativeArt.md
@@ -3,7 +3,7 @@
 This is a guide to create a program that randomly generates shapes to display.
 
 At the end of this guide you will:
-*  Have used the [`graphics`](https://docs.rs/ggez/0.7.0/ggez/graphics/) module of `ggez` to draw shapes
+*  Have used the [`graphics`](https://docs.rs/ggez/0.8.0-rc0/ggez/graphics/) module of `ggez` to draw shapes
 
 ## Project Setup
 
@@ -42,36 +42,36 @@ Test to make sure everything is correct by running `cargo run`.
 
 If there are no errors and you see a window you are good.
 
-## [ggez::graphics](https://docs.rs/ggez/0.7.0/ggez/graphics/)
+## [ggez::graphics](https://docs.rs/ggez/0.8.0-rc0/ggez/graphics/)
 
-Glancing over the docs for [`ggez::graphics`](https://docs.rs/ggez/0.7.0/ggez/graphics/) you can see there is a lot of functionality there.
-The basic shapes can be found in [`graphics::Mesh`](https://docs.rs/ggez/0.7.0/ggez/graphics/struct.Mesh.html#methods)
+Glancing over the docs for [`ggez::graphics`](https://docs.rs/ggez/0.8.0-rc0/ggez/graphics/) you can see there is a lot of functionality there.
+The basic shapes can be found in [`graphics::Mesh`](https://docs.rs/ggez/0.8.0-rc0/ggez/graphics/struct.Mesh.html#methods)
 
 So which shapes are in `Mesh`?
 
-* [line](https://docs.rs/ggez/0.7.0/ggez/graphics/struct.Mesh.html#method.new_line)
-* [circle](https://docs.rs/ggez/0.7.0/ggez/graphics/struct.Mesh.html#method.new_circle)
-* [ellipse](https://docs.rs/ggez/0.7.0/ggez/graphics/struct.Mesh.html#method.new_ellipse)
-* [polygon](https://docs.rs/ggez/0.7.0/ggez/graphics/struct.Mesh.html#method.new_polygon)
-* [polyline](https://docs.rs/ggez/0.7.0/ggez/graphics/struct.Mesh.html#method.new_polyline)
-* [rectangle](https://docs.rs/ggez/0.7.0/ggez/graphics/struct.Mesh.html#method.new_rectangle)
-* [rounded rectangle](https://docs.rs/ggez/latest/ggez/graphics/struct.Mesh.html#method.new_rounded_rectangle)
+* [line](https://docs.rs/ggez/0.8.0-rc0/ggez/graphics/struct.Mesh.html#method.new_line)
+* [circle](https://docs.rs/ggez/0.8.0-rc0/ggez/graphics/struct.Mesh.html#method.new_circle)
+* [ellipse](https://docs.rs/ggez/0.8.0-rc0/ggez/graphics/struct.Mesh.html#method.new_ellipse)
+* [polygon](https://docs.rs/ggez/0.8.0-rc0/ggez/graphics/struct.Mesh.html#method.new_polygon)
+* [polyline](https://docs.rs/ggez/0.8.0-rc0/ggez/graphics/struct.Mesh.html#method.new_polyline)
+* [rectangle](https://docs.rs/ggez/0.8.0-rc0/ggez/graphics/struct.Mesh.html#method.new_rectangle)
+* [rounded rectangle](https://docs.rs/ggez/0.8.0-rc0/ggez/graphics/struct.Mesh.html#method.new_rounded_rectangle)
 
 We are just going to touch on 2 shapes in this guide: the circle and the rectangle.
 
 Additionally, we need to look at another structure that we'll draw our things onto: the `Canvas`.
 We'll focus on three of its functions that are used to create it, draw a `Mesh` onto it and finally submit these draw calls:
 
-* [from_frame](https://docs.rs/ggez/latest/ggez/graphics/struct.Canvas.html#method.from_frame)
-* [draw](https://docs.rs/ggez/latest/ggez/graphics/struct.Canvas.html#method.draw)
-* [finish](https://docs.rs/ggez/latest/ggez/graphics/struct.Canvas.html#method.finish)
+* [from_frame](https://docs.rs/ggez/0.8.0-rc0/ggez/graphics/struct.Canvas.html#method.from_frame)
+* [draw](https://docs.rs/ggez/0.8.0-rc0/ggez/graphics/struct.Canvas.html#method.draw)
+* [finish](https://docs.rs/ggez/0.8.0-rc0/ggez/graphics/struct.Canvas.html#method.finish)
 
 ### ⚫ The Circle
 
 Circles are represented by 2 pieces of information: origin, and radius.
 Geometry, it's all coming back now.
 
-Here is the code for a [circle](https://docs.rs/ggez/0.7.0/ggez/graphics/struct.Mesh.html#method.new_circle):
+Here is the code for a [circle](https://docs.rs/ggez/0.8.0-rc0/ggez/graphics/struct.Mesh.html#method.new_circle):
 ```rust,skt-expression,no_run
 graphics::Mesh::new_circle(
     ctx,
@@ -89,7 +89,7 @@ Why did I just write like a million?!
 Well, `ctx` is needed to tell `ggez` where you are drawing to.
 `ctx` is what is passed to `update` and `draw` and holds our engine's state.
 
-[`graphics::DrawMode::fill()`](https://docs.rs/ggez/0.7.0/ggez/graphics/enum.DrawMode.html) is choosing between
+[`graphics::DrawMode::fill()`](https://docs.rs/ggez/0.8.0-rc0/ggez/graphics/enum.DrawMode.html) is choosing between
 outlining the circle or filling it in.
 
 Point, now here is one we expected.
@@ -132,7 +132,7 @@ If you see a circle on the screen when you run `cargo run` you're good!
 
 Rectangles are represented by 3 pieces of information: origin, width, and height.
 
-Here is the code for a [rectangle](https://docs.rs/ggez/0.7.0/ggez/graphics/struct.Mesh.html#method.new_rectangle):
+Here is the code for a [rectangle](https://docs.rs/ggez/0.8.0-rc0/ggez/graphics/struct.Mesh.html#method.new_rectangle):
 ```rust,skt-expression,no_run
 graphics::Mesh::new_rectangle(
     ctx,
@@ -145,8 +145,8 @@ graphics::Mesh::new_rectangle(
 It might seem weird that there are less parameters for more required information than a circle,
 but this is correct.
 
-[`Rect`](https://docs.rs/ggez/0.7.0/ggez/graphics/struct.Rect.html) is a convenient type that represents a... rectangle!
-[`graphics::Rect::new(500.0, 250.0, 200.0, 100.0)`](https://docs.rs/ggez/0.7.0/ggez/graphics/struct.Rect.html) positions the rectangle's top-left corner at `x: 500, y: 250` and
+[`Rect`](https://docs.rs/ggez/0.8.0-rc0/ggez/graphics/struct.Rect.html) is a convenient type that represents a... rectangle!
+[`graphics::Rect::new(500.0, 250.0, 200.0, 100.0)`](https://docs.rs/ggez/0.8.0-rc0/ggez/graphics/struct.Rect.html) positions the rectangle's top-left corner at `x: 500, y: 250` and
 specifies `width: 200, height: 100`.
 
 And that's how a rectangle is created!
@@ -279,7 +279,7 @@ We want 2 shapes randomly generated!
 Include your favorite RNG in your `Cargo.toml`:
 ```toml
 [dependencies]
-ggez = "0.7"
+ggez = "0.8.0-rc0"
 oorandom = "11"
 ```
 
@@ -342,6 +342,6 @@ Print it out, put it on a canvas, and sell it to your local museum.
 ## 💪 Challenges
 
 Optional activities for those that want more:
- * Randomize colors
- * Include more shapes: lines, polygons, ellipse
- * Draw a new shape every 5 seconds instead of upfront
+* Randomize colors
+* Include more shapes: lines, polygons, ellipse
+* Draw a new shape every 5 seconds instead of upfront

--- a/docs/guides/HelloGgez.md
+++ b/docs/guides/HelloGgez.md
@@ -35,7 +35,7 @@ version = "0.1.0"
 authors = ["Awesome Person awesome@person.com"]
 
 [dependencies]
-ggez = "0.7"
+ggez = "0.8.0-rc0"
 ```
 
 ### ✔ Check Project Setup
@@ -73,11 +73,11 @@ Game loops are responsible for:
 1. Updating state such as player position, health, etc.
 1. Drawing shapes, images, etc.
 
-`ggez` provides an [`EventHandler`](https://docs.rs/ggez/0.7.0/ggez/event/trait.EventHandler.html) as the default recommended interface to its internal loop to use in our games. Thanks `ggez`!
+`ggez` provides an [`EventHandler`](https://docs.rs/ggez/0.8.0-rc0/ggez/event/trait.EventHandler.html) as the default recommended interface to its internal loop to use in our games. Thanks `ggez`!
 
-You might have noticed [`EventHandler`](https://docs.rs/ggez/0.7.0/ggez/event/trait.EventHandler.html) is a Rust [Trait](https://doc.rust-lang.org/book/ch10-02-traits.html).
+You might have noticed [`EventHandler`](https://docs.rs/ggez/0.8.0-rc0/ggez/event/trait.EventHandler.html) is a Rust [Trait](https://doc.rust-lang.org/book/ch10-02-traits.html).
 This means it is intended to be implemented on a struct.
-There are quite a few callbacks defined on the Trait, but only [2 are required: update and draw](https://docs.rs/ggez/0.7.0/ggez/event/trait.EventHandler.html#required-methods).
+There are quite a few callbacks defined on the Trait, but only [2 are required: update and draw](https://docs.rs/ggez/0.8.0-rc0/ggez/event/trait.EventHandler.html#required-methods).
 
 Let's add `EventHandler` to our `src/main.rs` file:
 ```rust,skt-definition,no_run
@@ -99,9 +99,9 @@ You'll notice a couple new things here: `State`, `Context`, and `GameResult`.
 These could be player positions, scores, cards in your hand, etc..
 What is included in your state is very dependent on the game you are making.
 
-[`Context`](https://docs.rs/ggez/0.7.0/ggez/struct.Context.html) is how `ggez` gives you access to hardware such as mouse, keyboard, timers, graphics, sound, etc..
+[`Context`](https://docs.rs/ggez/0.8.0-rc0/ggez/context/struct.Context.html) is how `ggez` gives you access to hardware such as mouse, keyboard, timers, graphics, sound, etc..
 
-[`GameResult`](https://docs.rs/ggez/0.7.0/ggez/error/type.GameResult.html) is a utility provided by `ggez` to signify if there was an error or not.
+[`GameResult`](https://docs.rs/ggez/0.8.0-rc0/ggez/error/type.GameResult.html) is a utility provided by `ggez` to signify if there was an error or not.
 Internally it's just a `Result<(), GameError>`, which is why we implement `EventHandler` with `GameError`, so that [`on_error`](https://docs.rs/ggez/0.7.0/ggez/event/trait.EventHandler.html#method.on_error) knows what to expect.
 But, we're not going to write any bugs right? 😉
 
@@ -150,7 +150,7 @@ Also, don't mind the loop does nothing right now.
 We'll get to that soon.
 
 Now is the time for us to interface with our hardware and do something fun.
-To do that, you need to create a [`Context`](https://docs.rs/ggez/0.7.0/ggez/struct.Context.html) courtesy of `ggez`.
+To do that, you need to create a [`Context`](https://docs.rs/ggez/0.8.0-rc0/ggez/context/struct.Context.html) courtesy of `ggez`.
 
 Add this to the end of your `main` fn:
 ```rust,skt-expression,no_run
@@ -162,8 +162,8 @@ let (ctx, event_loop) = ContextBuilder::new("hello_ggez", "awesome_person")
 ```
 
 This will create a `Context` with the `game_id` `hello_ggez` and the author `awesome_person`.
-It will also create an [`EventLoop`](https://docs.rs/ggez/0.7.0/ggez/event/struct.EventLoop.html).
-We'll need it in a minute to call [`run`](https://docs.rs/ggez/0.7.0/ggez/event/fn.run.html).
+It will also create an [`EventLoop`](https://docs.rs/ggez/0.8.0-rc0/ggez/event/struct.EventLoop.html).
+We'll need it in a minute to call [`run`](https://docs.rs/ggez/0.8.0-rc0/ggez/event/fn.run.html).
 Feel free to replace the author with yourself.
 You are awesome after all.
 
@@ -224,7 +224,7 @@ let state = State {
 ```
 
 So now that we have state to update, let's update it in our `update` callback!
-We'll use [`timer::delta`](https://docs.rs/ggez/0.7.0/ggez/timer/fn.delta.html) to get the delta time.
+We'll use [`TimeContext::delta`](https://docs.rs/ggez/0.8.0-rc0/ggez/timer/struct.TimeContext.html#method.delta) to get the delta time.
 
 ```rust,skt-update,no_run
 fn update(&mut self, ctx: &mut Context) -> GameResult {

